foo/bar.js

```diff
@@ -0,0 +1,15 @@
+// This is a sample file for screenshot purposes
+
+
+export function isChicken(term: string): boolean {
+    if (term == "chicken") {
+	return true;
+    }
```

Comment:
chore:

Can we switch this to use triple equals `===` ?

```suggestion
    if (term === "chicken") {
	return true;
    }
```